packages/web3-utils/test/fixtures/validation.ts

```diff
@@ -0,0 +1,275 @@
+import { Numbers } from '../../src/types';
+
+export const hexStrict: [any, boolean][] = [
+	['0x48', true],
+	['0x123c', true],
+	['0x0dec0518fa672a70027b04c286582e543ab17319fbdd384fa7bc8f3d5a542c0b', true],
+	['0xd115bffabbdd893a6f7cea402e7338643ced44a6', true],
```

Comment:
```suggestion
	['0xd115bffabbdd893a6f7cea402e7338643ced44a6', true],
	['0x2C941171bD2A7aEda7c2767c438DfF36EAaFdaFc', true],
```

Adding a checksummed ETH address